spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraRelation.scala

```diff
@@ -42,7 +42,7 @@ private[cassandra] case class CassandraRelation
 
 object ColumnDataType {
 
-  private val primitiveTypeMap = Map[connector.types.ColumnType[_], types.DataType](
```

Comment:
One more thins - is it possible to create custom types in Catalyst for types like `UUIDType` and `InetType`? If so, could you create a separate ticket to address this?